src/cookie-consent.mjs

```diff
@@ -49,8 +49,6 @@ export default class Dialog extends HTMLElement {
     // initialize show and hide
     this.show = this.show();
     this.hide = this.hide();
-    // get all preferences
-    this.preferences.getAll();
```

Comment:
The `getAll()`  functions returns preferences. Running it without assigning it to a variable doesn't do anything.